sdk/appconfiguration/Azure.Data.AppConfiguration/README.md

```diff
@@ -220,7 +220,7 @@ This project has adopted the Microsoft Open Source Code of Conduct. For more inf
 [azconfig_asof_snapshot]: https://docs.microsoft.com/en-us/azure/azure-app-configuration/concept-point-time-snapshot
 [source_root]: https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/appconfiguration/Azure.Data.AppConfiguration
 [source_samples]: https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/appconfiguration/Azure.Data.AppConfiguration/samples
-[reference_docs]: https://azure.github.io/azure-sdk-for-net/api/Azure.Data.AppConfiguration.html
+[reference_docs]: https://azuresdkdocs.blob.core.windows.net/$web/dotnet/Azure.Data.AppConfiguration/1.0.0-preview.4/api/index.html
```

Comment:
Are these final? Seems odd we're linking to blob.core.windows.net and with what looks like a replacement variable `$web`.